docs/naming.md

```diff
@@ -1,86 +1,149 @@
-# Imagery naming conventions
+# Imagery Naming Conventions
 
-Imagery is named according to the majority region that imagery covers while also adhering to a few naming conventions.
+Imagery dataset titles and S3 paths are constructed from metadata about each imagery survey so that they will be consistent and human readable. Imagery is stored according to the majority region that each dataset covers.
 
-## Imagery Path
+## Imagery Dataset Titles
+
+The imagery dataset title is constructed from metadata that is entered when an imagery dataset is processed.
+
+```
+<geographic_description> <gsd>m [<geospatial_category>|<survey_number>] (<start_year>[-<end_year>?])[ - <lifecycle>?]
```

Comment:
Is there a way to format this to say `geographic_description` or `region`? 
(geographic_description is optional so if it isn't provided the region name will be used)

Reply:
Add clarification on this